### ISSUE TITLE
ShEx support?

### ISSUE THREAD

**Issue by ajs6f** [AKSW/RDFUnit#92]
Hi, just a quick question here: is AKSW interested in ShEx support in the future? (I could not any mention in the documentation I read: my apologies if I missed it and am asking a question that has already been answered.)

**Comment by jimkont** [comment]
yes, this is on the future plans (but indeed not yet written anywhere besides oral discussions).
I was thinking to integrate with ShExJava or Shaclex instead on a first iteration but haven't got the time to look into that yet

**Comment by ajs6f** [comment]
Good to hear, thanks! ShExJava now uses Apache Commons RDF, which would give you a bit of flexibility, but I don't know how interesting that is to you.

**Comment by jimkont** [comment]
Thanks for the heads up!
Another goal is to natively support RDF4J (in addition to Jena) so need to look at Apache Commons RDF anyway.

I would like to keep this issue open as a reminder and for others to discover easier so I am reopening it.
